### ISSUE TITLE
VictoryGroup causes SVG parse error in IE11

### ISSUE THREAD

**Issue by samkelleher** [FormidableLabs/victory#438]
Including this as a heads up incase someone else runs into the same issue, got the error but solved it by gradually removing parts of the chart until it worked to narrow down the culprit. The chart has animation and tooltips etc, but in the end go it to work by removing `VictoryGroup`.

A chart with
```
<VictoryGroup data={myData}>
  <VictoryLine  />
</VictoryGroup>
```

will throw an exception in IE11 and not render (the chart will render fine, just no line).

> SVG4601: SVG Path data has incorrect format and could not be completely parsed.

The workaround is to remove the group:

```
<VictoryLine data={myData}  />
```

**Comment by boygirl** [comment]
cannot reproduce.

**Comment by generalui-johnny5** [comment]
@samkelleher It worked!
I had the same problem you described, I removed the VictoryGroup component and it worked like a charm!
Gracias

**Comment by icd2k3** [comment]
I can repro this issue as well in latest version of Victory - switching from <VictoryGroup> to <VictoryChart> fixes it...

**Comment by icd2k3** [comment]
Edit: @boygirl I think this definitely has to do with using javascript date objects for a data axis that causes this issue.

**Comment by Luis-Jovel** [comment]
@icd2k3 @samkelleher 
Guys, I found the issue with VictoryGroup and IE/Edge.

As of the current version (20.0.0), the source file for [victory-group.js](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-group/victory-group.js) tries to create a new Date object in a way that is not supported on IE/Edge.

`victory-chart/src/components/victory-group/victory-group.js` on line 170

```
const _x1 = datum._x instanceof Date ? new Date(datum._x + xOffset) : datum._x + xOffset;
return assign({}, datum, { _x1 });
```

The code that breaks in IE is that offset addition `new Date(datum._x + xOffset)`, so I modified that line like this:

```
const _x1 = datum._x instanceof Date ? (isIE ? new Date(datum._x) : new Date(datum._x + xOffset)) : datum._x + xOffset;
return assign({}, datum, {_x1});
```

And it worked for me. I may create a pull request later on.
